src/configLoader.js

```diff
@@ -13,12 +14,20 @@ export function configLoader(configPath) {
         return process.exit(1);
     }
 
+    const rawConfig = fs.readFileSync(configPath, 'utf8');
+    const compiledConfig = rawConfig.replace(/\$\{([A-Za-z_]+)\}\$/g, (match, variableName) => {
```

Comment:
I'd still like to allow numbers in here. My thinking is that if you define an environment variable with a number and use it in your config file, kongfig will silently use the unsubstituted value, which could be pretty difficult to debug. If you feel strongly about not allowing digits, I'd at least like to find a way to make sure some kind of message is printed explaining that the value wasn't substituted.

Reply:
okay, i think i've solved both concerns here:

1. allow numbers
2. separate identifying the templated variable from validating the variable name - still only accepting specific characters and exiting if it's invalid, but capturing all values enclosed in `${}`